### ISSUE TITLE
Timing numbers reported in the tables

### ISSUE THREAD

**Issue by zubair-irshad** [jkulhanek/nerfbaselines#21]
Thanks for the great repo. I am sure this would be useful for a baselining various methods. Can you please share how did you report the timing numbers (on which GPUs/Batch Size?). Also I presume these are training numbers, is my understanding correct?

For mipnerf360 on TNT couthouse scene, I am getting an estimated completion time of `42 hours` with `1.61 it/s`. I am running it on 4 `Tesla-V100 16GB` GPUs. Does this seem reasonable, and any ideas on how could I speed it up? Thanks for your help! 

![image](https://github.com/user-attachments/assets/758199d6-4c5f-4c19-93be-7b889558adbb)

**Comment by jkulhanek** [comment]
Hi,
For the reported GPU numbers:
1) All hyperparameters are taken from the original datasets. The reported GPU mem is the peek GPU mem usage during training. Note, that this may not be the size of the model as some methods (ingp) load all images into GPU.
2) The GPU time is the total training time. It will be shorter than the total time `nerfbaselines train` will take as `nerfbaselines train` performs other things like checkpointing, train/test set evaluation, metrics reporting, etc. The `Time` is calculated as the total time taken by all training iterations only.

42 hrs for MipNeRF 360 seems reasonable to me. MipNeRF360 is really slow and takes a ton of memory. Also, I was using faster GPUs when I trained it on the MipNeRF 360 dataset so I think it is plausible.

If you finish training MipNeRF 360 on TT dataset, would you mind contributing the trained checkpoints (output artifacts)? It would be a great addition!

**Comment by jkulhanek** [comment]
Also, you can reduce the frequency of eval iterations by adding `--eval-few-iters ::10000` for example.

**Comment by zubair-irshad** [comment]
Great, thanks for the intuition. My current plan is to only benchmark it on the courthouse scene, will open up a PR once I have other scenes trained on mipnerf360, sometime in the future. 

Closing as resolved.
